runtime/runtime-core/jvm/test/aws/smithy/kotlin/runtime/retries/impl/AdaptiveRetryIntegrationTestResources.kt

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package aws.smithy.kotlin.runtime.retries.impl
+
+import aws.smithy.kotlin.runtime.retries.policy.RetryErrorType
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+val adaptiveRetryCubicTestCases = mapOf(
```

Comment:
Can you add the source of these test cases? Not a direct link but just a hint for future reference